lib/if_rmap.c

```diff
@@ -333,3 +333,15 @@ void if_rmap_terminate(void)
 		return;
 	list_delete(&if_rmap_ctx_list);
 }
+
```

Comment:
So with this change I have a ctx that is not associated with any vrf, How do we plan to get this re-associated with the vrf when it is added back in?

Reply:
I use a name pointer that can be either a vrf name or something else.